proxy-recorder/launch-gatling-proxy.sh

```diff
@@ -169,6 +169,27 @@ read
 
 FIND_COMMAND='find ./user-files/bodies -name *.txt'
 FIND_COUNT=`${FIND_COMMAND} |wc -l`
+  
+# The find count will return the number of things found, which will
+# be greater than 1 when there are "leftover" results in the directories
+# due to past aborts (like with ^C).  Issue a warning about needing
+# a little house cleaning, then try to recover and continue on 
+#
+if [ ${FIND_COUNT} -gt 1 ]	# Use numeric comparison, not string
+then
+   echo "
+WARNING!
+There are left-over recording bodies in the \"./user-files/bodies\" directory 
+and the \"../simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/\"
+directory (.txt and .scala files).  If the recording information for them is complete, 
+then you can manually copy the existing files to the proper directories or delete them 
+and re-run this recording if necessary.
+
+Attempting to continue.
+"
+  FIND_COUNT=1			# Force completion for the current recording
```

Comment:
I think I agree with @camlow325 that it would be better to bail out rather than pollute a recording. The idea of a `--force` flag sounds good. Alternately, would it be sufficient to just improve the error message, or keep the message above as an error rather than a warning, but not attempt to continue? At least that way there'd be an error with instructions on how to fix it.